test/groovy/com/sap/piper/analytics/TelemetryTest.groovy

```diff
@@ -134,12 +142,15 @@ class TelemetryTest extends BasePiperTest {
             stepParam1: 'something'
         ])
         // asserts
-        assertThat(jscr.shell, hasItem(containsString('curl -G -v "https://webanalytics.cfapps.eu10.hana.ondemand.com/tracker/log"')))
-        assertThat(jscr.shell, hasItem(containsString('--data-urlencode "action_name=Piper Library OS"')))
-        assertThat(jscr.shell, hasItem(containsString('--data-urlencode "event_type=library-os"')))
-        assertThat(jscr.shell, hasItem(containsString('--data-urlencode "custom3=anyStep"')))
-        assertThat(jscr.shell, hasItem(containsString('--data-urlencode "custom4=1234"')))
-        assertThat(jscr.shell, hasItem(containsString('--data-urlencode "custom5=abcd"')))
-        assertThat(jscr.shell, hasItem(containsString('--data-urlencode "custom11=something"')))
+        assertThat(httpParams, is(not(null)))
+        assertThat(httpParams?.url.toString(), allOf(
```

Comment:
what is the reason for the `?` after `httpParams`?

Reply:
leftover, this is now covered by the assert the line above